domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3000,13 +3074,22 @@ public void getConnection(Handler<AsyncResult<PgConnection>> replyHandler) {
     });
   }
 
-  void getSQLConnection(Handler<AsyncResult<SQLConnection>> handler) {
+   void getSQLConnection(Handler<AsyncResult<SQLConnection>> handler) {
+    getSQLConnection(handler, 0);
+   }
+
+  void getSQLConnection(Handler<AsyncResult<SQLConnection>> handler, int executionTimeLimit) {
```

Comment:
Can you add javadoc what unit (hours, minutes, ...) the timeout parameter has, and what 0 means?

Reply:
The parameter ordering makes it difficult to read when calling this method.
Can you put the handler at the end?
`getSQLConnection(int timeout, Handler<AsyncResult<SQLConnection>> handler)`